spec/features/remove_overrides_spec.rb

```diff
@@ -34,4 +34,9 @@
     # app/overrides/cells/decidim/meetings/highlighted_meetings_for_component_cell_decorator.rb
     expect(Decidim.version).to be < "0.22"
   end
+
+  it "remove: Fix accept invitation to private meetings" do
+    # app/overrides/cells/decidim/meetings/upcoming_events_cell_decorator.rb
+    expect(Decidim.version).to be < "0.22"
```

Comment:
The first version that has the fix is 0.23 via https://github.com/decidim/decidim/pull/6779
```suggestion
    expect(Decidim.version).to be < "0.23"
```